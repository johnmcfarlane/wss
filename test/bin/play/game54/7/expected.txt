 50  1,8  | AQUA
 50  1,8  | AQUO
 37  8,7  | AQUAE
 36  8,7  | AQUA
 36  8,7  | AQUO
 24  3,5  | ANOA
 21 11,3  | SEQUA
 20  4,8  - CAQUE
 20  4,8  - COQUE
 19  2,8  - SACQUE
 19  2,8  - SECQUE
 18  3,9  | EAU
 18 11,3  | SEQ
 17  1,2  | AA
 16  1,7  | EAU
 15  1,3  | AA
 15  1,3  | AE
 15  4,8  - COQ
 15  3,6  | NOA
 15  4,1  | QERE
 14  1,8  | EAU
 14  3,8  | EAU
 13  1,7  - ARUI
 13 11,2  | OSE
 13  9,7  - SUQ
 13  1,5  - UKE
 12  3,8  | AA
 12  3,8  | AE
 12  1,5  - AKA
 12  1,5  - AKO
 12  3,8  | EE
 12  1,5  - EKE
 12  3,8  | EO
 12  8,7  - ESQ
 12  3,8  | OE
 12  1,5  - OKA
 12  1,5  - OKE
 12  9,7  - SEQ
 11  3,1  | AA
 11  3,1  | AE
 11  3,5  | AN
 11  5,2  - AWA
 11  5,2  - AWE
 11  8,6  | EAU
 11  3,1  | EE
 11  3,1  | EO
 11  5,2  - EWE
 11  2,5  - KA
 11  7,3  | LOA
 11  7,3  | LOE
 11  3,1  | OE
 11  5,2  - OWE
 11  8,5  - QI
 10  5,4  - ESU
 10  2,4  - NAE
 10  2,4  - NEE
  9  1,1  | AA
  9  4,1  | AARU
  9  1,2  - EBOE
  9  7,1  | EEL
  9 11,3  | SEA
  9 11,3  | SEE
  8  1,8  | AA
  8  1,8  | AE
  8  4,2  | ARU
  8  2,2  - BAA
  8  2,2  - BAE
  8  2,2  - BEA
  8  2,2  - BEE
  8  2,2  - BOA
  8  4,8  - COUE
  8  8,7  | EAU
  8  5,9  | EAU
  8  1,8  | EE
  8  7,2  | EL
  8 10,6  | EO
  8  1,8  | EO
  8  5,5  - EPA
  8  5,5  - EPE
  8  3,6  | NO
  8  1,8  | OE
  8  5,5  - OPA
  8  5,5  - OPE
  8  2,7  - ROI
  8  2,8  - SACAE
  8  6,2  - WE
  7  1,2  - ABU
  7  5,2  | AE
  7  4,1  | AURA
  7  5,2  - AW
  7  4,8  - CUE
  7  5,2  | EE
  7  4,1  | EURE
  7  4,1  | EURO
  7  5,2  - EW
  7  5,2  | OE
  7  5,5  | OO
  7  5,5  - OP
  7  5,2  - OW
  7  6,5  - PEAI
  7  2,8  - SACO
  7  2,8  - SOCE
  6  1,3  - AA
  6  1,7  | AA
  6  8,7  | AA
  6  3,9  | AA
  6  1,2  - ABA
  6  1,2  - ABO
  6  1,7  | AE
  6  8,7  | AE
  6  3,9  | AE
  6  4,1  | AERO
  6  2,2  - BU
  6  4,8  - CEE
  6  4,8  - COE
  6  9,1  - DOA
  6  9,1  - DOE
  6  9,1  - DOO
  6  9,2  - EDO
  6  1,7  | EE
  6  8,7  | EE
  6  3,9  | EE
  6  5,9  | EE
  6  1,7  | EO
  6  8,7  | EO
  6  3,9  | EO
  6  5,9  | EO
  6  1,2  - OBA
  6  1,2  - OBE
  6  1,7  | OE
  6  8,7  | OE
  6  3,9  | OE
  6  5,9  | OE
  6 11,2  | OS
  6  2,8  - SAC
  6  2,8  - SEC
  6  2,8  - SOC
  6  4,2  | URA
  6  4,2  | URO
  6  9,6  - eAU
  5  8,5  | AA
  5  8,6  | AA
  5  1,2  - AB
  5  8,5  | AE
  5  8,6  | AE
  5  4,2  | ARE
  5  4,2  | ARO
  5  2,2  - BA
  5  2,2  - BE
  5  2,2  - BO
  5  4,9  - EAU
  5  8,6  | EE
  5  8,6  | EO
  5  4,2  | ERA
  5  4,2  | ERE
  5  8,7  - ESAU
  5  7,3  | LO
  5  8,5  | OE
  5  8,6  | OE
  5  4,2  | ORA
  5  4,2  | ORE
  5  6,5  - PA
  5  6,5  - PE
  5  6,5  - PO
  5  6,4  - SO
  5  8,1  - UDO
  5  1,4  - UNA
  5  8,7  - USEE
  4  8,1  - ADO
  4  1,4  - ANA
  4  1,4  - ANE
  4  8,7  - ASEA
  4  7,7  - AUS
  4  7,7  - EASE
  4  8,1  - EDO
  4  5,3  | EE
  4  5,3  | EO
  4  5,4  - ES
  4  8,7  - ESU
  4  4,7  - IO
  4  2,4  - NU
  4 10,7  | OE
  4  1,4  - ONA
  4  1,4  - ONE
  4  5,6  | OO
  4  5,4  - OS
  4  4,1  | OUR
  4  4,3  | RE
  4  9,7  - SAU
  4  9,7  - SOU
  4  9,7  - SUE
  4  1,4  - UN
  4  8,7  - USE
  3  7,7  - AAS
  3  3,9  - AE
  3  4,1  | AER
  3  7,7  - AES
  3  1,4  - AN
  3  4,1  | AOR
  3  4,1  | EAR
  3  3,9  - EE
  3  4,9  - EE
  3  4,1  | EER
  3  1,4  - EN
  3  4,9  - EO
  3  8,7  - ESE
  3  2,4  - NA
  3  2,4  - NE
  3  2,4  - NO
  3  4,1  | OAR
  3  3,9  - OE
  3  4,1  | OER
  3  7,7  - OES
  3  1,4  - ON
  3  8,7  - OSE
  3  9,7  - SAA
  3  9,7  - SAE
  3  9,7  - SAO
  3  9,7  - SEA
  3  9,7  - SEE
  3  9,7  - SOE
  3  8,7  - US
  3  1,8  - US
  2  8,5  - AI
  2  4,2  | AR
  2  1,7  - AR
  2  8,7  - AS
  2  1,8  - AS
  2  4,2  | ER
  2  1,7  - ER
  2  8,7  - ES
  2  1,8  - ES
  2  8,5  - OI
  2  4,2  | OR
  2  1,7  - OR
  2  8,7  - OS
  2  1,8  - OS
  2  9,7  - SO
  2  9,6  - eE
  2  9,6  - eO
  1  8,6  - Ae
  1  8,6  - Ee
  1  8,6  - Oe
