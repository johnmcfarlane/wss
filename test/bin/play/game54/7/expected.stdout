 50  1,8  | AQUA
 37  8,7  | AQUAE
 36  8,7  | AQUA
 32 10,7  | AQUAE
 30 10,7  | AQUA
 19  2,8  - SACQUE
 18  3,9  | EAU
 17  1,2  | AA
 17  3,5  | ANA
 16  1,7  | EAU
 15  1,3  | AA
 15  1,3  | AE
 15  4,8  - COQ
 15  3,6  | NAE
 14  1,8  | EAU
 14  3,8  | EAU
 13  9,7  - SUQ
 13  1,5  - UKE
 12  3,8  | AA
 12  3,8  | AE
 12 10,6  | EAU
 12  3,8  | EE
 12  1,5  - EKE
 12  3,8  | EO
 12  8,7  - ESQ
 12  1,5  - OKA
 12  1,5  - OKE
 11  3,1  | AA
 11  3,1  | AE
 11  3,5  | AN
 11  5,2  - AWA
 11  5,2  - AWE
 11  8,6  | EAU
 11  3,1  | EE
 11  3,1  | EO
 11  5,2  - EWE
 11  2,5  - KA
 11  3,1  | OE
 11  5,8  | OE
 11  5,2  - OWE
 11  8,5  - QI
 10  2,4  - NAE
 10  2,4  - NEE
  9  1,1  | AA
  9  7,1  | EEL
  8  1,8  | AA
  8  1,8  | AE
  8  2,2  - BAA
  8  2,2  - BEE
  8  2,2  - BOA
  8  8,7  | EAU
  8  5,9  | EAU
  8  1,8  | EE
  8  7,2  | EL
  8 10,6  | EO
  8  1,8  | EO
  8  5,5  - EPE
  8  3,6  | NA
  8  1,8  | OE
  8  5,5  - OPE
  8  2,7  - RAI
  8  6,2  - WE
  7  5,2  | AE
  7  4,1  | AURA
  7  5,2  - AW
  7  4,8  - CUE
  7  5,2  | EE
  7  4,1  | EURO
  7  5,2  - EW
  7  5,2  | OE
  7  5,5  | OO
  7  5,5  - OP
  7  5,2  - OW
  7  2,8  - SOCA
  6  1,3  - AA
  6  1,7  | AA
  6  8,7  | AA
  6  3,9  | AA
  6  1,2  - ABA
  6  1,2  - ABO
  6  1,7  | AE
  6  8,7  | AE
  6  3,9  | AE
  6  4,1  | AERO
  6  4,8  - CEE
  6  9,1  - DOE
  6  1,7  | EE
  6  8,7  | EE
  6  3,9  | EE
  6  5,9  | EE
  6  1,7  | EO
  6  8,7  | EO
  6  3,9  | EO
  6  5,9  | EO
  6  1,2  - OBA
  6  1,2  - OBE
  6  1,7  | OE
  6  8,7  | OE
  6  3,9  | OE
  6  5,9  | OE
  6  2,8  - SAC
  6  2,8  - SEC
  6  9,6  - eAU
  5  8,5  | AA
  5  8,6  | AA
  5  1,2  - AB
  5  8,5  | AE
  5  8,6  | AE
  5  4,2  | ARE
  5  2,2  - BA
  5  2,2  - BE
  5  2,2  - BO
  5  4,8  - CO
  5  4,9  - EAU
  5  8,6  | EE
  5  8,6  | EO
  5  4,2  | ERA
  5  4,2  | ERE
  5  7,3  | LA
  5  7,3  | LO
  5  8,5  | OE
  5  8,6  | OE
  5  4,2  | ORA
  5  4,2  | ORE
  5  6,5  - PA
  5  6,5  - PE
  5  6,5  - PO
  5  6,4  - SA
  5  6,4  - SO
  5  8,1  - UDO
  4 10,7  | AA
  4  8,1  - ADO
  4 10,7  | AE
  4  1,4  - ANA
  4  1,4  - ANE
  4  8,7  - ASEA
  4  7,7  - EASE
  4  5,3  | EE
  4  5,3  | EO
  4  5,4  - ES
  4  2,4  - NU
  4 10,7  | OE
  4  1,4  - ONE
  4  5,4  - OS
  4  4,1  | OUR
  4  4,3  | RE
  4  9,7  - SAU
  4  9,7  - SOU
  4  9,7  - SUE
  4  1,4  - UN
  4  8,7  - USE
  3  7,7  - AAS
  3  3,9  - AE
  3  1,4  - AN
  3  4,1  | EAR
  3  3,9  - EE
  3  4,9  - EE
  3  1,4  - EN
  3  4,9  - EO
  3  8,7  - ESE
  3  2,4  - NA
  3  2,4  - NE
  3  2,4  - NO
  3  4,1  | OAR
  3  3,9  - OE
  3  7,7  - OES
  3  1,4  - ON
  3  8,7  - OSE
  3  9,7  - SAE
  3  9,7  - SEA
  3  9,7  - SEE
  3  8,7  - US
  3  1,8  - US
  2  8,5  - AI
  2  4,2  | AR
  2  1,7  - AR
  2  8,7  - AS
  2  1,8  - AS
  2  4,2  | ER
  2  1,7  - ER
  2  8,7  - ES
  2  1,8  - ES
  2  8,5  - OI
  2  4,2  | OR
  2  1,7  - OR
  2  8,7  - OS
  2  1,8  - OS
  2  9,7  - SA
  2  9,7  - SO
  2  9,6  - eE
  2  9,6  - eO
  1  8,6  - Ae
  1  8,6  - Ee
  1  8,6  - Oe
